modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/TypeScriptNodeClientCodegen.java

```diff
@@ -1,5 +1,6 @@
 package io.swagger.codegen.languages;
 
+import io.swagger.codegen.*;
```

Comment:
It's bad practice to use wildcard import .

Reply:
+1, agree

Reply:
+1, agree